packaging/rpm/SPECS/apache-ignite.spec

```diff
@@ -25,14 +34,113 @@ BuildArch:        noarch
 Apache Ignite™ is the in-memory computing platform composed of a strongly
 consistent distributed database with powerful SQL, key-value and processing APIs
 
+This package is a virtual aggregation package for all packages for current
+Apache Ignite release - install it to install all other packages
+
+
+%package -n %{nameCore}
+Summary:          Apache Ignite execution core 
+Version:          %{version}
+Group:            Development/System
+License:          ASL 2.0
+URL:              https://ignite.apache.org
+Obsoletes:        apache-ignite < 2.5.0
+Requires:         java-1.8.0, pkgconfig
+Requires(pre):    shadow-utils
+Provides:         %{name}-core
+AutoReq:          no
+AutoProv:         no
+BuildArch:        noarch
+%description -n %{nameCore}
+Apache Ignite's core files and libs, necessary for node operation
+
 
+%package -n %{nameLibs}
+Summary:          Apache Ignite optional libs
```

Comment:
Why no word "optional" in package name then?

Reply:
Optional is meant "by default" when you have package that is not required for the core.